zproject/config.py

```diff
@@ -2,10 +2,8 @@
 import os
 from typing import Optional, overload
 
-from django.core.exceptions import ImproperlyConfigured
 
-
-class ZulipSettingsError(ImproperlyConfigured):
+class ZulipSettingsError(Exception):
```

Comment:
Can you extend the commit message for this change to talk about its motivation and impact?